docs/source/release_notes.rst

```diff
@@ -10,7 +10,9 @@ Release Notes
         * Added calls to ``_handle_nullable_types`` in component fit, transform, and predict methods when needed :pr:`4046`, :pr:`4043`
         * Removed existing nullable type handling across AutoMLSearch to just use new handling :pr:`4085`, :pr:`4043`
         * Handled nullable type incompatibility in ``Decomposer`` :pr:`4105`, :pr:`4043`
+        * Removed nullable type incompatibility handling for ARIMA and ExponentialSmoothingRegressor :pr:`4129`
         * Changed the default value for ``null_strategy`` in ``InvalidTargetDataCheck`` to ``drop`` :pr:`4131`
+        * Pinned sktime version to 0.17.0 for nullable types support :pr:`4137`
```

Comment:
Is this being pinned specifically for nullable types support? I thought it was more general? Or is it because we upgraded to 0.17.0 for nullable type support that we now have to `==` instead of `<=` `0.17.0`?

Reply:
The latter is correct, yeah. Pinned to `==` to maintain nullable type support.

Reply:
got it! I think this thread is good enough explanation, then, if anyone is looking at this in the future (hi!)